cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -33,14 +33,16 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
       "CREATE ROLE databaseAdminUsers",
       "CREATE ROLE noAccessUsers",
       "CREATE ROLE roleAdder",
+      "CREATE ROLE roleNameModifier",
       "CREATE ROLE roleDropper",
       "CREATE ROLE roleAssigner",
       "CREATE ROLE roleRemover",
       "CREATE ROLE roleShower",
       "CREATE ROLE roleManager",
       "CREATE ROLE userAdder",
-      "CREATE ROLE userDropper",
+      "CREATE ROLE userNameModifier",
       "CREATE ROLE userModifier",
+      "CREATE ROLE userDropper",
```

Comment:
why not either keep it after adder and before modifier or put it after all modifiers? putting it in between feels odd